VALUES.md

```diff
@@ -0,0 +1,66 @@
+# Ayo.js Community Values
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at
+creating a new foundation of project governance and management that brings
+**humans** and their concerns front and center.
+
+You can pronounce it however you want: be it “ey-yo”, “I-oh”, or “Awooooo” — the
+Ayo.js project strives to create an environment where you can feel at home and
+focus on your contributions to this nice shared space that we all benefit from.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations,
+package management, libuv, V8 and Chakra, builds/ops, testing and its
+infrastructure, hardware and IoT, Android, streams, community events,
+facilitation, moderation and education. These and more are all concerns that
+can be handled separately and by members who might be much better qualified
+for these individual tasks than a central decision body.
+
+Ayo.js will go further than Node.js did in spreading responsibility across
+the project, so everyone can focus and specialize better than before.
+Contributors should be able to organize and manage their own subprojects
+in a way that is effective and efficient for their unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance
+policy should change with the required agility. Following the governance policy
+to the letter will not take precedence over staying true to our stated values or
+the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with
+members with all sorts of perspectives. You are bound to find yourself in
```

Comment:
The second person is used only in two places in the document. Here, and for the 'pronounciation' paragraph. I think it would be better to rewrite those two sections to use third person/passive like the rest of the document, for consistent style.

Reply:
Yeah, definitely agree, good to have that consistent. I'll go ahead and fix those since it's a small change, if it's no good feel free to revert/change.